pcs/lib/resource_agent/ocf_transform.py

```diff
@@ -170,3 +194,14 @@ def _bool_value(value: Optional[str]) -> bool:
     value -- raw bool value
     """
     return value == "1"
+
+
+def _bool_value_1_0(value: Optional[str]) -> bool:
```

Comment:
Naming of `_bool_value` vs `_bool_value_1_0` could be misleading as `1_0` in the name could mean ocf 1.0 or `1` and `0` as only possible bool values. Therefore I would suggest to rename `_bool_value_1_0` to `_bool_value_ocf_1_0` or to `_bool_value_legacy` as it doesn't completely conform to ocf 1.0.

Reply:
ack, changed to `_bool_value_legacy`